redeem/Redeem.py

```diff
@@ -687,6 +689,7 @@ def signal_handler(signal, frame):
 
     # Register signal handler to allow interrupt with CTRL-C
     signal.signal(signal.SIGINT, signal_handler)
+    signal.signal(signal.SIGTERM, signal_handler)
```

Comment:
I made it global because the printer "self" was not correct on the kill. As for the name, it's just a personal preference to try to be more explicit. Back in the old days, we needed to save every character that we could. But that often produced cryptic names. Today, the "cost" computation is far different.